swap_meet/vendor.py

```diff
@@ -0,0 +1,112 @@
+
+#wave_1
+class Vendor:
+    def __init__(self, inventory = None):
+        if inventory is None:
+            self.inventory = []
+        else: 
+            self.inventory = inventory 
+        
+    def adding_to_inventory(Vendor): #checks item in inventory & returns True or False 
+        if item in inventory:
+            self.inventory.add(item)
+        else:
+            self.inventory.remove(item)
+        return False 
+    
+    def add(self, item): #append item in inventory 
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item): #remove item from inventory
+        if item in self.inventory:
+            self.inventory.remove(item)
+        else:
+            return False 
```

Comment:
Consider refactoring to address the edge case first:
```suggestion
        if item not in self.inventory:
            return False
        
        self.inventory.remove(item)
```